project/src/main/webapp/document.jsp

```diff
@@ -311,11 +313,36 @@
       }
 
       // On comment click
-      $(document).on('click','.comment',function() {
+      $(document).on('click','.comment',function(event) {
         // Do real stuff
-        console.log("comment clicked");
+        var element = event.target.className;
+        console.log(element);
```

Comment:
currently updating this to highlight selected comment instead of just logging it